packages/runtime/container-runtime/src/orderedClientElection.ts

```diff
@@ -335,10 +402,16 @@ export class OrderedClientElection
     private addClient(client: ILinkedClient, sequenceNumber: number): void {
         if (this.isEligibleFn(client)) {
             this._eligibleCount++;
-            if (this._electedClient === undefined) {
-                // Automatically elect latest client
+            const isSummarizer = client.client.details.type === summarizerClientType;
```

Comment:
nit: Consider calling this isNewClientSummarizer or isAddedClientSummarizer for readability.